# The current state of a kinematic chain (joint angles)
Header header
string name
string[] joint_names
float64[] position_measured
float64[] position_desired

